types/bosch/parkingLotSensor0_39/meta.json

```diff
@@ -0,0 +1,14 @@
+{
+  "name": "Parking Lot Sensor V0.39.2",
```

Comment:
it might make sense to name it V0.39.X in order to keep it up to date for minor improvements